integration/ct_prep_test.sh

```diff
@@ -36,7 +36,7 @@ for ((i=0; i < RPC_SERVER_COUNT; i++)); do
   RPC_SERVERS="${RPC_SERVERS},localhost:${port}"
 
   echo "Starting Log RPC server on port ${port}"
```

Comment:
Discussed in person.

I'll make sure the messages match the flag format (ie, we should log *host:port*, not just *port*). We didn't reach a clear decision on the endpoint vs uri debate, so I'll go ahead with endpoint for now. I do prefer it slightly over uri, but I'm happy to change if people think otherwise.